src/helpers.rs

```diff
@@ -140,7 +145,13 @@ pub(crate) fn positions_to_faces(
             .iter()
             .map(|original_face|
                 // With vertex faces in left-hand order.
-                index_of(original_face, face_index).unwrap() as VertexKey)
+                match index_of(original_face, face_index){
+                    Some(i) => i,
```

Comment:
Again, use `unwrap_or()`. Clippy will probably also warn you that the return after the `assert!(false, ...)` is dead code.
That warning will go away then.

Reply:
The asserts were from stress tests and should have been removed.  Thanks for spotting, I forgot those were in there.

This is the one spot unwrap_or can be used (the other matches return a value for the None case).